### ISSUE TITLE
update github location for codewind/che clusterRole

### ISSUE THREAD

**Issue by AndyRWatson** [eclipse/codewind#1604]
**Codewind version:**   0.7
**OS:** N/A

**Che version:**
**IDE extension version:**
**IDE version:**
**Kubernetes cluster:**

**Description of the enhancement:**

This is a request to look at improving our installation process for the CHE and particularly the page https://www.eclipse.org/codewind/mdt-che-installinfo.html

In this page we reference the latest clusterrole for che/codewind an that url also includes the version number of the plugin to use.  Though we've updated the docs to reflect the plugin version changes I wondered if there was anything we can do here so that the url always the same and points to the latest version of the plugin.   My concern here is if we run the risk of not updating the page in a future and  have  our install instructions  pointing people to an outdated releases of our che plugin ?

**Proposed solution:** 

1. change our release process so the url for the cluster role remains constant,   e.g.

https://raw.githubusercontent.com/eclipse/codewind-che-plugin/setup/install_che/codewind-clusterrole.yaml

instead of

https://raw.githubusercontent.com/eclipse/codewind-che-plugin/**0.7.0**/setup/install_che/codewind-clusterrole.yaml

2. update our documentation page to be the constant url 

https://raw.githubusercontent.com/eclipse/codewind-che-plugin/setup/install_che/codewind-clusterrole.yaml

Thoughts ?

**Comment by johnmcollier** [comment]
So, I've had some thoughts on this too.

We used to point to the roles on the master branch of the repo (https://raw.githubusercontent.com/eclipse/codewind-che-plugin/master/setup/install_che/codewind-clusterrole.yaml), but since the roles could change from release to release, we changed it to the corresponding release branch. The problem is that this means we have to make sure we update that doc every release, and I believe we even forgot one release (luckily the roles didn't change then!).

I wonder if we could have the Jenkins build for each release branch publish the cluster role on downloads.eclipse.org (or somewhere else), and ensure it's at a constant link?

**Comment by AndyRWatson** [comment]
maybe as part of the release we could publish on codewind-resources ?

**Comment by johnmcollier** [comment]
Yeah, that could work too, and may be simpler than messing with Jenkins.
